### PR TITLE
automatic display of deeplink controller removed from testbed apps - AIS-127

### PR DESCRIPTION
**Pull request by parthkalavadia**
Automatic display of deeplink controller is removed and is handled manually in the block of initSession in Appdelegate of obj-c and swift.

### CODE REVIEW

**Comment by BranchMacMini** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/qabot.stage.branch.io/b388335146db7e936b7215574d83b172/b388335146db7e936b7215574d83b22a)
---
*Duration*: 4 minutes and 24 seconds
*Result*: **Perfect build!** All 169 tests passed. :+1:
*Test Coverage*: 57%

File: Branch-TestBed-Swift/TestBed-Swift/AppDelegate.swift [hunk 1]
```diff
@@ -36,35 +45,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 DataStore.setActivePendingSetDebugEnabled(false)
             }
             
-            let navigationController = UIStoryboard(name: "Main", bundle: nil).instantiateInitialViewController() as! UINavigationController
-            branch.registerDeepLinkController(navigationController, forKey:"~referring_link")
-            
             // Required. Initialize session. automaticallyDisplayDeepLinkController is optional (default is false).
-            branch.initSession(launchOptions: launchOptions, automaticallyDisplayDeepLinkController: true, deepLinkHandler: { params, error in
+            branch.initSession(launchOptions: launchOptions, automaticallyDisplayDeepLinkController: false, deepLinkHandler: { params, error in
```
Comment (aaustin) [inline Branch-TestBed-Swift/TestBed-Swift/AppDelegate.swift]: let's change to use a the different initSession method that doesn't have `automaticallyDisplayDeepLinkController` in the arguments. Can you remove it altogether?

File: Branch-TestBed/Branch-TestBed/AppDelegate.m [hunk 2]
```diff
@@ -38,28 +38,27 @@ - (BOOL)application:(UIApplication *)application
...
+            [[UIStoryboard storyboardWithName:@"Main" bundle:[NSBundle mainBundle]]
+                instantiateInitialViewController];
+    
+        [branch registerDeepLinkController:navigationController forKey:@"deeplink_text"];
+     */
 
     // Required. Initialize session. automaticallyDisplayDeepLinkController is optional (default is NO).
     [branch initSessionWithLaunchOptions:launchOptions
-        automaticallyDisplayDeepLinkController:YES
+        automaticallyDisplayDeepLinkController:NO
```
Comment (aaustin) [inline Branch-TestBed/Branch-TestBed/AppDelegate.m]: same comment here

**Comment by parthkalavadia** [comment]
Replaced the method call with
`- (void)initSessionWithLaunchOptions:(NSDictionary *)options andRegisterDeepLinkHandler:(callbackWithParams)callback`

**Comment by BranchMacMini** [comment]
Result of [Integration 2](https://stlt.herokuapp.com/v1/xcs_deeplink/qabot.stage.branch.io/b388335146db7e936b7215574d83b172/91a3adb793e487e2a3459e6632f53979)
---
*Duration*: 3 minutes and 41 seconds
*Result*: **Perfect build!** All 169 tests passed. :+1:
*Test Coverage*: 57%

**Comment by aaustin** [comment]
👍

**Pull request merged**